src/lib.rs

```diff
@@ -45,6 +45,7 @@ pub enum HoverPosition {
 /// Registers a panel with drag reordering for a given ID.
 pub fn use_drag_reorder<E>(
     id: impl Into<Oco<'static, str>>,
+    transpose: bool,
```

Comment:
Should `transpose` be on `use_drag_reorder`? Or on `provide_drag_reorder`? It  seems like the transposing should be done at the provide level?